sparkle/utils/aws.py

```diff
@@ -0,0 +1,18 @@
+import boto3
+
+
+def get_paths(bucket='msds-sparkle', max_files=1000):
+    session = boto3.Session(
+        aws_access_key_id='AKIAW7CYB6L5QMHLLHUU',
+        aws_secret_access_key='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'
```

Comment:
@collinprather Need to read in the access keys from a **local** `credentials.csv` file (which *can't* be uploaded to the repo).